### PR TITLE
New builder: Aggregate

### PR DESCRIPTION
**Pull request by andresmgot**
New section "Aggregate" which generates a summarize expression:

Screenshot from 2022-09-09 16-16-22

Notes:
 - Aggregate functions usually requires a column to apply it to (unless count)
 - Percentile is the only function that requires an additional parameter (percentile number)

### CODE REVIEW

File: src/components/QueryEditor/VisualQueryEditor/AggregateItem.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                  },
+                ],
+                type: QueryEditorExpressionType.Reduce,
+              });
+          }}
+        />
+      )}
+      {aggregate.reduce?.name !== AggregateFunctions.Count && (
+        <>
+          <Label style={{ margin: '9px 9px 0 9px' }}>of</Label>
```
Comment (asimpson) [inline src/components/QueryEditor/VisualQueryEditor/AggregateItem.tsx:99]: Do we have this pattern else-where? The `of` feels disconnected to me. Might be worth running by UX?

Comment (asimpson) [inline src/components/QueryEditor/VisualQueryEditor/AggregateItem.tsx:99]: Do we have this pattern else-where? The `of` feels disconnected to me. Might be worth running by UX?

**Review by asimpson — APPROVED** [review]
Nice work on this! :trophy:

File: src/components/QueryEditor/VisualQueryEditor/AggregateItem.tsx (hunk 1 above)
Comment (andresmgot) [reply src/components/QueryEditor/VisualQueryEditor/AggregateItem.tsx:99]: Good shout, for the moment I am only mimicking whatever was already in the previous editor but I am taking note about this as a specific question for UX

**Pull request merged**